### ISSUE TITLE
getDisplayedNotifications returning wrong ids

### ISSUE THREAD

**Issue by alloystory** [notifee/react-native-notifee#381]
to reproduce:
```
await notifee.displayNotification({
    id: 'abc',
    title: 'title',
    body: 'body',
})

console.log(await notifee.getDisplayedNotifications())
```

expected:
```
[{
  id: 'abc',
  date: '1630835976663',
  notification: {
      id: 'abc',
      title: 'title',
      body: 'body',
  },
}]
```

actual:
```
[{
  id: '3556498',
  date: '1630835976663',
  notification: {
      id: '3556498',
      title: 'title',
      body: 'body',
  },
}]
```

this happened on Android 11; i didnt test this scenario on other platforms.

**Comment by adar2378** [comment]
Same issue. I can not cancel those notifications using the returned ids :|

**Comment by mikehardy** [comment]
Oh no, that doesn't seem to match expectations. We've got :eyes: on it, thank you for reporting @alloystory and confirming @adar2378 - need to investigate if this is usage issue or a bug as a first pass, and if an error, correct of course

**Comment by andidev** [comment]
@adar2378 
however cancelling works with using the original id `notifee.cancelNotification('abc')`

**Comment by andidev** [comment]
Same problem occurs if string is a number string `123`

**Comment by adar2378** [comment]
@andidev  yes, but for unassigned ids it is impossible to detect and cancel the notification by using getDisplayedNotification API

**Comment by helenaford** [comment]
Hi all, this is now fixed in release v3.0.0 (release notes [here](https://notifee.app/react-native/docs/release-notes#300))
